modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,94 @@ def main():
+                            help='Specify the path to parse PELs')
+    else:
+        parser.add_argument('-A', '--archive', action='store_true',
+                            help='Fetch PEL(s) details from the archived path')
 
-    parser.add_argument('-f', '--file', dest='file',
-                        help='input pel file to parse')
-    parser.add_argument('-s', '--serviceable',
-                        help='Parse serviceable (not info/recovered) PELs',
-                        action='store_true')
-    parser.add_argument('-N', '--non-serviceable',
-                        help='Parse non-serviceable (info/recovered) PELs',
-                        action='store_true')
     parser.add_argument('-P', '--skip-parser-plugins', dest='skip_plugins',
-                        action='store_true', help='Skip loading plugins')
-    parser.add_argument('-j', '--json',
-                        help='Process all files in a directory and save as <filename>.json.'
-                        ' Use -o to specify output directory',
-                        action='store_true')
-    parser.add_argument('-o', '--output-dir', dest='output_dir',
-                        help='Directory to write output files when processing a directory')
-    parser.add_argument('-e', '--extension', dest='extension',
-                        help='Only look for files with this extension (e.g. ".pel")')
-    parser.add_argument('-c', '--clean', dest='clean', action='store_true',
-                        help='Delete original file after parsing')
-    parser.add_argument('-t', '--termination', dest='critSysTerm',
-                        action='store_true', help='List only critical system terminating PELs')
-    parser.add_argument('-i', '--id',
-                        dest='pelID', help='Display a PEL based on its ID')
-    parser.add_argument('--bmc-id',
-                        dest='bmcID', help='Display a PEL based on its BMC Event ID')
-    parser.add_argument('--plid',
-                        dest='plID', help='Display PELs based on its Platform Log ID')
-    parser.add_argument('--src',
-                        dest='src', help='Display PELs based on its System Reference Code')
-    parser.add_argument('--src-exclude',
-                        dest='src_exclude_file', help='Display PELs excluding SRCs from the file')
-    parser.add_argument('-l', '--list',
-                        action='store_true', help='List PELs')
-    parser.add_argument('-a', '--all-pels', dest='all',
-                        action='store_true', help='Display all PELs')
+                        action='store_true', help='Skip loading PEL parser plugins')
+
+    parser.add_argument('-f', '--file', dest='file',
+                        metavar='</path/to/pel/file>',
+                        help='Input PEL file to extract PEL data')
+
+    parser.add_argument('-l', '--list', action='store_true',
+                        help='List PELs summary')
```

Comment:
I don't think we need 'summary' here; we want to keep the 'list' terminology to match the argument and the terminology people are used to.